### PR TITLE
Webapp reports successful transaction even if the sender is not a co-sginer

### PR DESCRIPTION
**Pull request by ashishnair-zil**
1.) Webapp reports successful transaction even if the sender is not a co-sginer
Jira no: https://zilliqa-jira.atlassian.net/jira/software/projects/US/boards/11?selectedIssue=US-98

### CODE REVIEW

File: src/smartcontract/multisig_wallet.scilla.js [hunk 1]
```diff
@@ -0,0 +1,407 @@
+const CODE = `scilla_version 0
```
Comment (inamirandaz) [inline src/smartcontract/multisig_wallet.scilla.js:1]: @ashishnair-zil is this change supposed to be here or this is from another PR/commit?

File: src/assets/style.scss [hunk 2]
```diff
@@ -402,7 +402,7 @@ p {
     display: flex;
 
     input {
-      width: 38px;
+      width: 52px;
```
Comment (inamirandaz) [inline src/assets/style.scss:405]: @ashishnair-zil this is not for this PR, right?

File: src/components/Wallet/NewTransaction.vue [hunk 3]
```diff
@@ -63,8 +63,8 @@
         <input
           type="text"
           class="form-control"
-          placeholder="Teg transition"
-          aria-label="Teg transition"
+          placeholder="Tag transition"
```
Comment (inamirandaz) [inline src/components/Wallet/NewTransaction.vue:66]: @ashishnair-zil this is not for this PR, right?

**Review by inamirandaz — CHANGES_REQUESTED** [review]
@ashishnair-zil can we remove the changes/commits not relevant to this PR?

**Comment by ashishnair-zil** [comment]
Closing this PR as [PR 29](https://github.com/Zilliqa/zilliqa-multisig-wallet/pull/29) include this changes

**Pull request closed without merging**